test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/ExtendedQueryTagTests.cs

```diff
@@ -47,17 +48,28 @@ public async Task GivenExtendedQueryTag_WhenReindexing_ThenShouldSucceed()
                 return;
             }
 
-            DicomTag weightTag = DicomTag.PatientWeight;
-            DicomTag sizeTag = DicomTag.PatientSize;
+            string tag1PrivateCreator = "Unit Test 1";
+            DicomTag tag1 = RandomDicomTag(tag1PrivateCreator);
+            string weightTagVRCode = DicomVR.DS.Code;
```

Comment:
I vote for updating the name instead of using the same just to keep things explicit. If you feel strongly about using just 1 variable, please feel free to comment and I will update that.